app/controllers/idv/sessions_controller.rb

```diff
@@ -31,7 +44,7 @@ def start_idv_session
     end
 
     def applicant_from_params
-      app_vars = applicant_params.delete_if { |_key, value| value.blank? }
+      app_vars = idv_params.merge(financial_params.delete_if { |_key, value| value.blank? })
```

Comment:
we are storing the applicant_params from the first page. We then merge them with the results of the second page and submit all the non-blank ones to the proofer.